src/js/index.js

```diff
@@ -138,7 +138,9 @@
     this._parseContent = parseContent
 
     // Destroy it if it exists first.
-    tocbot.destroy()
+    if (!options.skipRendering) {
```

Comment:
I think it should still be destroyed otherwise the scroll listeners could get created twice and you'd have multiple.

Reply:
you might want to put this around `document.querySelector(options.tocSelector).innerHTML = ''` though, not sure how you are handling dom generation / regeneration. We could add a callback to help with this piece if it'd be helpful.

Reply:
You're completely right, I was actually thinking about this in our project: https://github.com/SocialGouv/code-du-travail-numerique/pull/847/files#diff-df6e7465c13d67e297a1d84e73446f7dR41